full-service/src/util/import_from_v1.rs

```diff
@@ -0,0 +1,227 @@
+use reqwest::header::CONTENT_TYPE;
+use serde_json::{json, Value};
+use std::convert::TryFrom;
+
+pub fn import_accounts(conn: &Conn, client: &reqwest::blocking::Client, request_uri: &str) {
+    let body = json!({
+        "method": "get_all_accounts",
+        "jsonrpc": "2.0",
+        "id": 1
+    });
+
+    let response = send_request(client, request_uri, body);
+
+    let account_map = response.get("result").unwrap().get("account_map").unwrap();
+    let account_ids = response
+        .get("result")
+        .unwrap()
+        .get("account_ids")
+        .unwrap()
+        .as_array()
+        .unwrap();
+
+    for account_id in account_ids {
+        let account: AccountJSON = serde_json::from_value(
+            account_map
+                .get(account_id.as_str().unwrap())
+                .unwrap()
+                .clone(),
+        )
+        .unwrap();
+
+        import_account(&account, conn, request_uri, client);
+    }
+}
+
+fn import_account(
+    account: &AccountJSON,
+    conn: &Conn,
+    request_uri: &str,
+    client: &reqwest::blocking::Client,
+) {
+    let body = json!({
+        "method": "export_account_secrets",
+        "jsonrpc": "2.0",
+        "id": 1,
+        "params": {
+            "account_id": account.account_id
+        }
+    });
+
+    let response = send_request(client, request_uri, body);
+
+    let account_secrets: AccountSecrets = serde_json::from_value(
+        response
+            .get("result")
+            .unwrap()
+            .get("account_secrets")
+            .unwrap()
+            .clone(),
+    )
+    .unwrap();
+
+    if let Some(mnemonic) = account_secrets.mnemonic {
+        let mnemonic = Mnemonic::from_phrase(&mnemonic, bip39::Language::English).unwrap();
+        Account::import(
+            &mnemonic,
+            Some(account.name.clone()),
+            0,
+            Some(account.first_block_index.parse::<u64>().unwrap()),
+            Some(account.next_subaddress_index.parse::<u64>().unwrap()),
+            "".to_string(),
+            "".to_string(),
+            "".to_string(),
+            conn,
+        )
+        .unwrap();
+    } else if let Some(entropy) = account_secrets.entropy {
+        let entropy_bytes = hex::decode(entropy).unwrap();
+        let entropy = RootEntropy::try_from(entropy_bytes.as_slice()).unwrap();
+        Account::import_legacy(
+            &entropy,
+            Some(account.name.clone()),
+            0,
+            Some(account.first_block_index.parse::<u64>().unwrap()),
+            Some(account.next_subaddress_index.parse::<u64>().unwrap()),
+            "".to_string(),
+            "".to_string(),
+            "".to_string(),
+            conn,
+        )
+        .unwrap();
+    } else {
+        panic!(
+            "No entropy or mnemonic found for account {}",
+            account.account_id
+        );
+    };
+
+    let body = json!({
+        "method": "get_sent_transaction_logs_for_account",
```

Comment:
I don't see `get_sent_transaction_logs_for_account` anywhere else in the code. In this version of FS or in V1.9.1

Reply:
it's in v1.9.4+